docker-compose.geth-nimbus2.yml

```diff
@@ -0,0 +1,116 @@
+version: "3.8"
+
+services:
+
+  init-geth:
+    image: ethereum/client-go:$GETH_VERSION
+    container_name: init-geth
+    volumes:
+      - $EXECUTION_DATA_VOLUME:/execution_data
+      - $CONFIGS_VOLUME:/configs
+    command: >
+      --datadir=/execution_data init /configs/genesis.json
+
+  geth:
+    image: ethereum/client-go:$GETH_VERSION
+    container_name: geth
+    depends_on:
+      - init-geth
+    volumes:
+      - $EXECUTION_DATA_VOLUME:/execution_data
+      - $CONFIGS_VOLUME:/configs
+    restart: unless-stopped
+    stop_signal: SIGINT
+    stop_grace_period: 2m
+    command: >
+      --datadir=/execution_data
+      --ws
+      --ws.api "eth,net,web3"
+      --ws.addr 0.0.0.0
+      --ws.origins "*"
+      --http
+      --http.api "eth,net,web3"
+      --http.addr "0.0.0.0"
+      --http.corsdomain "*"
+      --http.vhosts "*"
+      --maxpeers 50
+      --ipcdisable
+      --authrpc.jwtsecret /configs/jwt.hex
+      --bootnodes $EXECUTION_BOOTSTRAP_NODE_1,$EXECUTION_BOOTSTRAP_NODE_2
+      --networkid $LUKSO_NETWORK_ID
+      --verbosity $GETH_VERBOSITY
+      --metrics
+      --metrics.expensive
+      --metrics.addr "0.0.0.0"
+      --miner.gaslimit 42000000
+      --miner.gasprice 1000000000
```

Comment:
Please change to default one: https://github.com/lukso-network/network-configs/blob/6474ce52fc62b347365b9726c29c0363bfff67d8/mainnet/geth/geth.toml#L11